Tests/Serilog.Exceptions.Test/Destructurers/UriDestructurerTest.cs

```diff
@@ -0,0 +1,66 @@
+namespace Serilog.Exceptions.Test.Destructurers
+{
+    using System;
+    using System.Collections;
+    using Core;
+    using Exceptions.Destructurers;
+    using Xunit;
+
+    public class UriDestructurerTest
```

Comment:
Personally, I would place the tests in one of the existing suites, preferable in the suite for reflection based destructurer. No other type of object has its own suite, Uri will stick out.

Reply:
Agreed.